fragment/fragment-lint/src/test/java/androidx/fragment/lint/UseGetLayoutInflaterTest.kt

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.fragment.lint
+
+import com.android.tools.lint.checks.infrastructure.LintDetectorTest
+import com.android.tools.lint.detector.api.Detector
+import com.android.tools.lint.detector.api.Issue
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+/* ktlint-enable max-line-length */
+/* ktlint-disable max-line-length */
+@RunWith(JUnit4::class)
+class UseGetLayoutInflaterTest : LintDetectorTest() {
+
+    override fun getDetector(): Detector = UseGetLayoutInflater()
+
+    override fun getIssues(): MutableList<Issue> = mutableListOf(UseGetLayoutInflater.ISSUE)
+
+    private val dialogFragmentCorrectImplementationStubJava = java(
+        """
+            package foo;
+            import android.os.Bundle;
+            import android.view.View;
+            import android.view.ViewGroup;
+            import androidx.annotation.NonNull;
+            import androidx.annotation.Nullable;
+            import androidx.fragment.app.DialogFragment;
+            
```

Comment:
Remove trailing white space here.

Reply:
Thanks for pointing it out, I appreciate this and sorry to bother you. But almost all other test stubs in this package were formatted like this.

Reply:
The format is correct, it is not the new lines it is the white space on the line itself. Are you able to run lintdebug and have it pass? Here are the exact errors we are seeing:

Warnings: 8

Lint
Check Contents - Please remove the trailing whitespace.
?Location: "fragment/fragment-lint/src/test/java/androidx/fragment/lint/UseGetLayoutInflaterTest.kt" line 132-132 col 66-67
Analyzer Description: Validates that changes do not have prohibited expressions in them.
Please remove the trailing whitespace.

Lint
Check Contents - Please remove the trailing whitespace.
?Location: "fragment/fragment-lint/src/test/java/androidx/fragment/lint/UseGetLayoutInflaterTest.kt" line 84-84 col 74-75
Analyzer Description: Validates that changes do not have prohibited expressions in them.
Please remove the trailing whitespace.

Lint
Check Contents - Please remove the trailing whitespace.
?Location: "fragment/fragment-lint/src/test/java/androidx/fragment/lint/UseGetLayoutInflaterTest.kt" line 85-85 col 74-75
Analyzer Description: Validates that changes do not have prohibited expressions in them.
Please remove the trailing whitespace.

Lint
Check Contents - Please remove the trailing whitespace.
?Location: "fragment/fragment-lint/src/test/java/androidx/fragment/lint/UseGetLayoutInflaterTest.kt" line 88-88 col 0-20
Analyzer Description: Validates that changes do not have prohibited expressions in them.
Please remove the trailing whitespace.

Lint
Check Contents - Please remove the trailing whitespace.
?Location: "fragment/fragment-lint/src/test/java/androidx/fragment/lint/UseGetLayoutInflaterTest.kt" line 90-90 col 87-88
Analyzer Description: Validates that changes do not have prohibited expressions in them.
Please remove the trailing whitespace.

Lint
Check Contents - Please remove the trailing whitespace.
?Location: "fragment/fragment-lint/src/test/java/androidx/fragment/lint/UseGetLayoutInflaterTest.kt" line 110-110 col 74-75
Analyzer Description: Validates that changes do not have prohibited expressions in them.
Please remove the trailing whitespace.

Lint
Check Contents - Please remove the trailing whitespace.
?Location: "fragment/fragment-lint/src/test/java/androidx/fragment/lint/UseGetLayoutInflaterTest.kt" line 111-111 col 74-75
Analyzer Description: Validates that changes do not have prohibited expressions in them.
Please remove the trailing whitespace.

Lint
Check Contents - Please remove the trailing whitespace.
?Location: "fragment/fragment-lint/src/test/java/androidx/fragment/lint/UseGetLayoutInflaterTest.kt" line 131-131 col 67-68
Analyzer Description: Validates that changes do not have prohibited expressions in them.
Please remove the trailing whitespace.

Reply:
lint passes on my PC otherwise I wouldn't open the PR or it could be the case that I'm missing something

Reply:
I guess I fixed all of them. but GH actions still fail, it works on my fork (only, navigation part fails for other reason)